poller-lambdas/src/config.ts

```diff
@@ -0,0 +1,23 @@
+import { fromNodeProviderChain } from '@aws-sdk/credential-providers';
+import { POLLER_LAMBDA_ENV_VAR_KEYS } from '../../shared/pollers';
+
+/**
+ * Is this application running locally, or in AWS?
+ * LAMBDA_TASK_ROOT & AWS_EXECUTION_ENV are set when running in AWS
+ * See: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
+ */
+export const isRunningLocally =
+	!process.env.LAMBDA_TASK_ROOT && !process.env.AWS_EXECUTION_ENV;
+
+export const remoteAwsConfig = isRunningLocally
+	? {
+			region: 'eu-west-1',
+			credentials: fromNodeProviderChain({
+				profile: 'editorial-feeds',
+			}),
+		}
+	: {};
+
+export const getEnvironmentVariableOrCrash = (
+	key: keyof typeof POLLER_LAMBDA_ENV_VAR_KEYS,
+) => process.env[key]!;
```

Comment:
kind of a nit: this won't crash if the var is unset, but it'll assert that it was set, and the caller may (or may not!) crash when it tries to use the var. it might be nicer to forcefully throw from here instead? which might make following the stack trace easier to follow

Reply:
wow! TIL that the 'bang' doesn't actually verify at runtime, but rather its cue from the developer to compile to assume it's not nullish 🫨 (I use it quite often and I need to stop). I've replaced it in https://github.com/guardian/newswires/pull/83/commits/f914a118e0a9250098982eedc1997491be191708